interface/main/messages/print_postcards.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * /library/MedEx/MedEx_background.php
+ *
+ * This file is executed as a background service
+ * either through ajax or cron.
+ *
+ * Copyright (C) 2017 MedEx <magauran@MedExBank.com>
+ *
+ * LICENSE: This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *  Portions of this were developed using Terry Hill's addr_label code.
+ *
+ * @package LibreHealth EHR
+ * @author MedEx <support@MedExBank.com>
+ * @link http:Librehealth.io
+ */
+
+$fake_register_globals=false;
+$sanitize_all_escapes=true;
+
+require_once("../../globals.php");
+require_once("$srcdir/fpdf/fpdf.php");
+require_once("$srcdir/formatting.inc.php");
+
+# This is based on session array.
+$pid_list = array();
+$pid_list = $_SESSION['pidList'];
+
+//$pdf = new FPDF('L', 'mm', array(297.638,  419.528));
+$pdf = new FPDF('L', 'mm', array(148,105));
+$last = 1;
+$pdf->SetFont('Arial','',14);
+//var_dump($_SESSION['pidList']);exit;
+
+#Get the data to place on labels
+#and output each label
+foreach ($pid_list as $pid) {
+$pdf->AddPage();
+$patdata = sqlQuery("SELECT " .
+  "p.fname, p.mname, p.lname, p.pubpid, p.DOB, " .
+  "p.street, p.city, p.state, p.postal_code, p.pid " .
+  "FROM patient_data AS p " .
+  "WHERE p.pid = ? LIMIT 1", array($pid));
+
+# sprintf to print data
+$text = sprintf("  %s %s\n  %s\n  %s %s %s\n ", $patdata['fname'], $patdata['lname'], $patdata['street'], $patdata['city'], $patdata['state'], $patdata['postal_code']);
+
+$postcard_message = "It's time to get your EYES checked!
+Please call our office to schedule\nyour eye exam  at (413) 276-4543.
```

Comment:
We need to work on the postcards. They will be configurable from the MedExbank server like the SMS, Email and AVM messages. We were working on these first and then get to the other features.